js/adapt-youtube.js

```diff
@@ -1,208 +1,206 @@
 
-      this.setIFrameSize();
-    },
+        this.debouncedTriggerGlobalEvent('play');// use debounced version because seeking whilst playing will trigger two 'play' events
 
-    onMediaStop: function(view) {
-      // if it was this view that triggered the media:stop event, ignore it
-      if (view && view.cid === this.cid) return;
+        this.isPlaying = true;
 
-      if (this.isPlaying) {
-        this.player.pauseVideo();
-      }
-    },
+        if (this.model.get('_setCompletionOn') && this.model.get('_setCompletionOn') === 'play') {
+          this.setCompletionStatus();
+        }
+      break;
+      case YT.PlayerState.PAUSED:
+        this.isPlaying = false;
 
-    onPlayerReady: function() {
-      if (this.model.get('_media')._playbackQuality) {
-        this.player.setPlaybackQuality(this.model.get('_media')._playbackQuality);
-      }
-    },
-
-    /**
-    * this seems to have issues in Chrome if the user is logged into YouTube (possibly any Google account) - the API just doesn't broadcast the events
-    * but instead throws the error:
-    * Failed to execute 'postMessage' on 'DOMWindow': The target origin provided ('https://www.youtube.com') does not match the recipient window's origin ('http://www.youtube.com').
-    * This is documented here:
-    *   https://code.google.com/p/gdata-issues/issues/detail?id=5788
-    * but I haven't managed to get any of the workarounds to work... :-(
-    */
-    onPlayerStateChange: function(e) {
-      switch(e.data) {
-        case YT.PlayerState.PLAYING:
-          Adapt.trigger('media:stop', this);
-
-          this.debouncedTriggerGlobalEvent('play');// use debounced version because seeking whilst playing will trigger two 'play' events
-
-          this.isPlaying = true;
-
-          if (this.model.get('_setCompletionOn') && this.model.get('_setCompletionOn') === 'play') {
-            this.setCompletionStatus();
-          }
-        break;
-        case YT.PlayerState.PAUSED:
-          this.isPlaying = false;
-
-          this.triggerGlobalEvent('pause');
-        break;
-        case YT.PlayerState.ENDED:
-          this.triggerGlobalEvent('ended');
-
-          if (this.model.get('_setCompletionOn') && this.model.get('_setCompletionOn') === 'ended') {
-            this.setCompletionStatus();
-          }
-        break;
-      }
-    },
+        this.triggerGlobalEvent('pause');
+      break;
+      case YT.PlayerState.ENDED:
+        this.triggerGlobalEvent('ended');
 
-    onSkipToTranscript: function() {
-      // need slight delay before focussing button to make it work when JAWS is running
-      // see https://github.com/adaptlearning/adapt_framework/issues/2427
-      _.delay(function() {
-        this.$('.youtube__transcript-btn').a11y_focus();
-      }.bind(this), 250);
-    },
+        if (this.model.get('_setCompletionOn') && this.model.get('_setCompletionOn') === 'ended') {
+          this.setCompletionStatus();
+        }
+      break;
```

Comment:
ESLint: wrong indentation.
```suggestion
        break;
```
See also the other `break` statements.